test/functional/toolchains/google/test_bazel.py

```diff
@@ -33,9 +39,9 @@ def base_profile():
 
 @pytest.mark.parametrize("build_type", ["Debug", "Release", "RelWithDebInfo", "MinSizeRel"])
 @pytest.mark.tool("bazel", "6.3.2")
-def test_basic_exe_6x(bazelrc, build_type, base_profile):
+def test_basic_exe_6x(bazelrc, build_type, base_profile, bazel_output_root_dir):
```

Comment:
Maybe we can just pass `client.current_folder/some_subdir` and save us the fixture and creating another separate folder?

Reply:
I used a fixture for this module to keep the same cache folder, so avoid using a different one for each test

Reply:
Actually, I tested both options and the time consuming is more or less the same